spyder/plugins/editor.py

```diff
@@ -1887,14 +1888,9 @@ def save(self, index=None, force=False):
         return editorstack.save(index=index, force=force)
     
     @Slot()
-    def save_as(self):
-        """Save *as* the currently edited file"""
-        editorstack = self.get_current_editorstack()
-        if editorstack.save_as():
-            fname = editorstack.get_current_filename()
-            if CONF.get('workingdir', 'editor/save/auto_set_to_basedir'):
-                self.open_dir.emit(osp.dirname(fname))
-            self.__add_recent_file(fname)

```

Comment:
@goanpeca I remove it because integrate all logic on Widget module, And only call save_as functionality from Widget module with this sentence... on line 1893 (plugins/editor.py)
> `self.get_current_editorstack().save_as(index=index)`